MeetupPlatformApi/Persistence/EntitiesConfiguration/MeetupEntityConfiguration.cs

```diff
@@ -38,5 +39,67 @@ public void Configure(EntityTypeBuilder<Meetup> meetupEntity)
             .Property(meetup => meetup.EndTime)
             .IsRequired()
             .HasColumnName("end_time");
+
+        meetupEntity
+            .HasOne<Organizer>()
+            .WithMany(organizer => organizer.Meetups)
+            .OnDelete(DeleteBehavior.SetNull)
+            .HasForeignKey(meetup => meetup.OrganizerId)
+            .HasConstraintName("fk_users_meetups_organizer_id");
+
+        meetupEntity
+            .Property(meetup => meetup.OrganizerId)
+            .IsRequired()
+            .HasDefaultValueSql("gen_random_uuid()")
+            .HasColumnName("organizer_id");
+
+        meetupEntity
+            .HasIndex(meetup => meetup.OrganizerId)
+            .HasName("ix_meetups_organizer_id");
+
+        meetupEntity
+            .HasMany(meetup => meetup.SignedUpUsers)
+            .WithMany(plainUser => plainUser.Meetups)
+            .UsingEntity<Dictionary<string, object>>(
+            "meetups_plain_users",
+            join => join
+                    .HasOne<PlainUser>()
+                    .WithMany()
+                    .HasForeignKey("user_id")
+                    .HasConstraintName("fk_meetups_plain_users_plain_users_user_id")
+                    .OnDelete(DeleteBehavior.Cascade),
+            join => join
+                    .HasOne<Meetup>()
+                    .WithMany()
+                    .HasForeignKey("meetup_id")
+                    .HasConstraintName("fk_meetups_plain_users_meetups_meetup_id")
+                    .OnDelete(DeleteBehavior.Cascade));
+
+        meetupEntity
```

Comment:
But why though? Couldn't You done it inside previous many-to-many configuration?